reconcile/utils/terrascript_aws_client.py

```diff
@@ -2451,6 +2455,49 @@ def populate_tf_resource_s3_cloudfront(self, resource, namespace_info):
             '${' + cf_oai_tf_resource.id + '}'
         tf_resources.append(Output(output_name_0_13, value=output_value))
 
+        # aws_s3_bucket_acl
+        values = common_values.get('distribution_config', {})
+        if 'logging_config' in values.keys():
+            values = {}
+            values['name'] = identifier
+            values['tags'] = common_values['tags']
+
+            aws_s3_bucket_acl_resource = aws_s3_bucket_acl(identifier, **values)
+            tf_resources.append(aws_s3_bucket_acl_resource)
+
+            access_control_policy = {
+                'grant': {
+                    'grantee': {
+                        'id': '${' + aws_s3_bucket_acl_resource.id + '}',
+                        'type': '${' + aws_s3_bucket_acl_resource.type + '}'
+                    },
+                    'permission': 'READ'
+                }
+            }
+            grant = {
+                'grantee': '${' + aws_s3_bucket_acl_resource.grantee + '}',
+                'permission': '${' + aws_s3_bucket_acl_resource.permission + '}'
+            }
+            owner = {
+                'id': '${' + aws_s3_bucket_acl_resource.id + '}'
+            }
+
+            values['access_control_policy'] = access_control_policy
+            values['grant'] = grant
+            values['owner'] = owner
+
+            aws_s3_bucket_acl_resource = aws_s3_bucket_acl(identifier, **values)
+            tf_resources.append(aws_s3_bucket_acl_resource)
+
+            output_name_0_13 = output_prefix + '__acl'
+            output_value = '${' + aws_s3_bucket_acl.acl + '}'
+            tf_resources.append(Output(output_name_0_13, value=output_value))
+
+            if self._multiregion_account(account):
+                values['provider'] = 'aws.' + region
```

Comment:
Here you're adding to the `values` which was defined for the `aws_s3_bucket_acl_resource` resource above. Is that what you want ?

Reply:
I am wondergin if this is needed at all. Should probably be removed all together
```
            if self._multiregion_account(account):
                values['provider'] = 'aws.' + region
            bucket_policy_tf_resource = aws_s3_bucket_policy(identifier, **values)
            tf_resources.append(bucket_policy_tf_resource)
```